src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,170 @@
 package core.basesyntax;
 
+//import jdk.internal.util.ArraysSupport;
+import java.util.Arrays;
+import java.util.NoSuchElementException;
```

Comment:
remove commented import
```suggestion
import java.util.Arrays;
import java.util.NoSuchElementException;
```